style.css

```diff
@@ -0,0 +1,66 @@
+:root{
+    --primary-color: #DAA06D;
+}
+
+*{
+    box-sizing: border-box;
+    font-family: Verdana, Geneva, Tahoma, sans-serif;
+}
+body{
+    margin: 0;
+    padding: 0;
+}
+/* avatar style */
+.first-avatar{
+    width: 100%;
+    max-width: 50px;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+}
+.second-avatar{
+    width: 100%;
+    max-width: 70px;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+}
+.third-avatar{
+    width: 100%;
+    max-width: 100px;
+    border-radius: 50%;
+    background-color: var(--primary-color);
+}
+
+/* badge style */
+.div-flex{
+    display: flex;
+}
+.div-flex p{
```

Comment:
Do not apply any properties directly to the body element or any other element. Make appropriate utility classes having those properties and use it in the HTML file accordingly. Because if you use the component library elsewhere then it might interfere with the styling and you might not get to see what you intend to

Here you can name it as div-flex-text and pass it to the paragraph element in html